### PR TITLE
fix(example): clearTimeout and useCallback was added on CopyButton

### PR DESCRIPTION
**Pull request by tbor00**
In this PR we worked on the correct cleaning of timeOut and useCallback was added

### CODE REVIEW

**Comment by tbor00** [comment]
@rnike and @dai-shi  can help me you review this PR?

File: examples/src/components/CopyButton.js [hunk 1]
```diff
@@ -7,13 +7,17 @@ of a separate button component and with the added utility
...
+  const handleCopy =  useCallback(()  => {
     copyToClipboard(code).then(() => {
       setIsCopied(true)
-      setTimeout(() => setIsCopied(false), 3000)
+      timer.current =  setTimeout(() => setIsCopied(false), 3000)
     })
-  }
+    return () => {
+      clearTimeout(timer.current)
+    }
```
[the remark this answers is not in the document]
Comment (tbor00) [reply examples/src/components/CopyButton.js]: i just fixed it

**Review by dai-shi — COMMENTED** [review]
It looks good to me.

Wait for @rnike 's review as I'm not 100% sure about the original concern.

**Review by rnike — APPROVED** [review]
LGTM

**Review by dai-shi — APPROVED** [review]
Let's merge then.

**Pull request merged**